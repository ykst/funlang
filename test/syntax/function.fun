fun f ()
    $print("f OK\n")
\_
    
fun g()
    $print("g OK\n")
    f()
\_

fun h(x,y)
    $print(x < y)
    $print("\n")
\_

g()
h(1,2)
h(2,1)
