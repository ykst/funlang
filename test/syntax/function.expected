g OK
f OK
true
false
